addons/hr/static/tests/m2x_avatar_employee_tests.js

```diff
@@ -115,339 +163,441 @@ QUnit.test("many2one_avatar_employee widget in kanban view", async function (ass
-    const [userId_1, userId_2] = pyEnv["res.users"].create([
-        { partner_id: partnerId_1 },
-        { partner_id: partnerId_2 },
-    ]);
-    const [employeeId_1, employeeId_2] = pyEnv["hr.employee.public"].create([
-        { user_id: userId_1, user_partner_id: partnerId_1 },
-        { user_id: userId_2, user_partner_id: partnerId_2 },
-    ]);
-    const avatarId_1 = pyEnv["m2x.avatar.employee"].create({
-        employee_ids: [employeeId_1, employeeId_2],
-    });
-    const views = {
-        "m2x.avatar.employee,false,form":
-            '<form><field name="employee_ids" widget="many2many_avatar_employee"/></form>',
-    };
-    const { openView } = await start({
-        mockRPC(route, args) {
-            if (args.method === "web_read") {
-                assert.step(`web_read ${args.model} ${args.args[0]}`);
-            }
-            if (args.method === "read") {
-                assert.step(`read ${args.model} ${args.args[0]}`);
-            }
-        },
-        serverData: { views },
-    });
-    await openView({
-        res_model: "m2x.avatar.employee",
-        res_id: avatarId_1,
-        views: [[false, "form"]],
-    });
-    assert.containsN(
-        document.body,
-        ".o_field_many2many_avatar_employee .o_tag",
-        2,
-        "should have 2 records"
-    );
-    assert.strictEqual(
-        document
-            .querySelector(".o_field_many2many_avatar_employee .o_tag img")
-            .getAttribute("data-src"),
-        `/web/image/hr.employee.public/${employeeId_1}/avatar_128`
-    );
+        assert.containsN(
+            document.body,
+            ".o_field_many2many_avatar_employee .o_tag",
+            2,
+            "should have 2 records"
+        );
+        assert.strictEqual(
+            document
+                .querySelector(".o_field_many2many_avatar_employee .o_tag img")
+                .getAttribute("data-src"),
+            `/web/image/hr.employee.public/${employeeId_1}/avatar_128`
+        );
 
-    await dom.click(
-        document.querySelector(".o_field_many2many_avatar_employee .o_tag .o_m2m_avatar")
-    );
-    await dom.click(
-        document.querySelectorAll(".o_field_many2many_avatar_employee .o_tag .o_m2m_avatar")[1]
-    );
-    // TODO: avatar card employee
-    assert.verifySteps([`web_read m2x.avatar.employee ${avatarId_1}`]);
-});
+        // Clicking on first employee's avatar
+        await click(
+            document.querySelector(".o_field_many2many_avatar_employee .o_tag .o_m2m_avatar")
+        );
+        await contains(".o_avatar_card");
+        await contains(".o_card_user_infos > span", { text: "Mario" });
+        await contains(".o_card_user_infos > a", { text: "Mario@partner.com" });
+        await contains(".o_card_user_infos > a", { text: "+45687468" });
+        assert.strictEqual(
+            document.querySelector(".o_avatar_card_buttons button").textContent,
+            " Send message "
```

Comment:
I think because of one of my suggestions in my previous comment, we will have to change that: 
```suggestion
            "Send message"
```